cosmoz-omnitable-styles.js

```diff
@@ -132,7 +132,33 @@ export default css`
 			--paper-font-caption_-_line-height: 18px;
 		}
 	}
+	cosmoz-omnitable-amount-range-input,
+	cosmoz-omnitable-number-range-input,
+	cosmoz-omnitable-date-range-input,
+	cosmoz-omnitable-time-range-input,
+	cosmoz-omnitable-datetime-range-input,
+	paper-input {
```

Comment:
It should work the same if you defined these styles for the header containing element.
```suggestion
	[selector-for-header-container] {
```

Please try it out.

Reply:
Included all in the .header-cell selector, and it works.